sdks/java/io/rabbitmq/src/main/java/org/apache/beam/sdk/io/rabbitmq/RabbitMqMessage.java

```diff
@@ -19,8 +19,8 @@
 
 import com.rabbitmq.client.AMQP;
 import com.rabbitmq.client.AMQP.BasicProperties;
+import com.rabbitmq.client.Delivery;
```

Comment:
In the 4.x client this was `QueueingConsumer.Delivery`. In 5.x this is a stand-alone class.